src/main/java/org/spdx/library/model/SpdxPackage.java

```diff
@@ -606,19 +603,13 @@ protected List<String> _verify(Set<String> verifiedIds, String specVersion) {
 		
 		// files depends on if the filesAnalyzed flag
 		try {
-			if (getFiles().size() == 0) {
-				if (filesAnalyzed) {
```

Comment:
I noticed we removed the check for files being present when files analyzed is true.

I went back and read the spec, and indeed there is no stated requirement that files be present - so this change is correct.